scene/gui/tab_bar.cpp

```diff
@@ -1373,7 +1377,8 @@ int TabBar::get_tab_width(int p_idx) const {
 		style = theme_cache.tab_disabled_style;
 	} else if (current == p_idx) {
 		style = theme_cache.tab_selected_style;
-	} else if (hover == p_idx) {
+		// Use the unselected style's width if the hovered one is shorter, to avoid an infinite loop when switching tabs with the mouse.
+	} else if (hover == p_idx and theme_cache.tab_hovered_style->get_minimum_size().width >= theme_cache.tab_unselected_style->get_minimum_size().width) {
```

Comment:
```suggestion
	} else if (hover == p_idx && theme_cache.tab_hovered_style->get_minimum_size().width >= theme_cache.tab_unselected_style->get_minimum_size().width) {
```

Reply:
While we're at it, why just width? Can we not use the currently drawn style's entire size and ignore that of the focus style?

Reply:
@AThousandShips GDScript brainrot. :zany_face:

@YuriSizov For one, the height doesn't matter at all. Second, the way tabs are drawn is by passing the style to `_draw_tab()`, and that function isn't aware of the hovering state, so that actually would add more complexity.

Reply:
Okay, I see that the height is controlled through min size, basically, which would indeed make things way too complicated. Well, we'll see if anyone is annoyed by it.